src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -26,79 +27,28 @@
-
-// This is an example WASM from the SDK that can put and delete arbitrary
-// contract data.
-// clang-format off
-// $ wasm-strip target-tiny/wasm32-unknown-unknown/release/example_contract_data.wasm 
-// $ xxd -i target-tiny/wasm32-unknown-unknown/release/example_contract_data.wasm
-// clang-format on
-std::vector<uint8_t> contractDataWasm{
-    0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00, 0x01, 0x11, 0x03, 0x60,
-    0x02, 0x7e, 0x7e, 0x01, 0x7e, 0x60, 0x01, 0x7e, 0x01, 0x7e, 0x60, 0x02,
-    0x7f, 0x7e, 0x00, 0x02, 0x0d, 0x02, 0x01, 0x6c, 0x01, 0x5f, 0x00, 0x00,
-    0x01, 0x6c, 0x01, 0x32, 0x00, 0x01, 0x03, 0x04, 0x03, 0x02, 0x00, 0x01,
-    0x05, 0x03, 0x01, 0x00, 0x10, 0x06, 0x19, 0x03, 0x7f, 0x01, 0x41, 0x80,
-    0x80, 0xc0, 0x00, 0x0b, 0x7f, 0x00, 0x41, 0x80, 0x80, 0xc0, 0x00, 0x0b,
-    0x7f, 0x00, 0x41, 0x80, 0x80, 0xc0, 0x00, 0x0b, 0x07, 0x31, 0x05, 0x06,
-    0x6d, 0x65, 0x6d, 0x6f, 0x72, 0x79, 0x02, 0x00, 0x03, 0x70, 0x75, 0x74,
-    0x00, 0x03, 0x03, 0x64, 0x65, 0x6c, 0x00, 0x04, 0x0a, 0x5f, 0x5f, 0x64,
-    0x61, 0x74, 0x61, 0x5f, 0x65, 0x6e, 0x64, 0x03, 0x01, 0x0b, 0x5f, 0x5f,
-    0x68, 0x65, 0x61, 0x70, 0x5f, 0x62, 0x61, 0x73, 0x65, 0x03, 0x02, 0x0a,
-    0xd1, 0x01, 0x03, 0x38, 0x01, 0x01, 0x7f, 0x23, 0x00, 0x41, 0x10, 0x6b,
-    0x22, 0x02, 0x24, 0x00, 0x20, 0x02, 0x20, 0x01, 0x37, 0x03, 0x08, 0x20,
-    0x00, 0x20, 0x02, 0x41, 0x08, 0x6a, 0x29, 0x03, 0x00, 0x22, 0x01, 0x37,
-    0x03, 0x08, 0x20, 0x00, 0x20, 0x01, 0x42, 0x0f, 0x83, 0x42, 0x09, 0x52,
-    0xad, 0x37, 0x03, 0x00, 0x20, 0x02, 0x41, 0x10, 0x6a, 0x24, 0x00, 0x0b,
-    0x57, 0x01, 0x02, 0x7f, 0x23, 0x00, 0x41, 0x20, 0x6b, 0x22, 0x02, 0x24,
-    0x00, 0x20, 0x02, 0x41, 0x10, 0x6a, 0x20, 0x00, 0x10, 0x02, 0x02, 0x40,
-    0x20, 0x02, 0x29, 0x03, 0x10, 0xa7, 0x0d, 0x00, 0x20, 0x02, 0x29, 0x03,
-    0x18, 0x20, 0x02, 0x20, 0x01, 0x10, 0x02, 0x20, 0x02, 0x29, 0x03, 0x00,
-    0xa7, 0x0d, 0x00, 0x20, 0x02, 0x29, 0x03, 0x08, 0x23, 0x00, 0x41, 0x10,
-    0x6b, 0x22, 0x03, 0x24, 0x00, 0x10, 0x00, 0x1a, 0x20, 0x03, 0x41, 0x10,
-    0x6a, 0x24, 0x00, 0x20, 0x02, 0x41, 0x20, 0x6a, 0x24, 0x00, 0x42, 0x05,
-    0x0f, 0x0b, 0x00, 0x0b, 0x3e, 0x01, 0x02, 0x7f, 0x23, 0x00, 0x41, 0x10,
-    0x6b, 0x22, 0x01, 0x24, 0x00, 0x20, 0x01, 0x20, 0x00, 0x10, 0x02, 0x20,
-    0x01, 0x29, 0x03, 0x00, 0xa7, 0x04, 0x40, 0x00, 0x0b, 0x20, 0x01, 0x29,
-    0x03, 0x08, 0x23, 0x00, 0x41, 0x10, 0x6b, 0x22, 0x02, 0x24, 0x00, 0x10,
-    0x01, 0x1a, 0x20, 0x02, 0x41, 0x10, 0x6a, 0x24, 0x00, 0x20, 0x01, 0x41,
-    0x10, 0x6a, 0x24, 0x00, 0x42, 0x05, 0x0b};
+std::vector<uint8_t>
+get_testdata(std::string const& filename)
+{
+    std::filesystem::path fnPath("testdata");
+    fnPath /= filename;
+    std::ifstream in(fnPath);
+    REQUIRE(in);
+    in.exceptions(std::ios::badbit);
+    return std::vector<uint8_t>{std::istreambuf_iterator<char>(in), {}};
+}
+
+std::vector<uint8_t>
+get_example_i32_wasm()
+{
+    return get_testdata("example_add_i32.wasm");
```

Comment:
We should include a comment that mentions where these file can be generated.

Reply:
Added